src/main/java/mate/academy/service/impl/ParserServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import mate.academy.model.FruitTransaction;
+import mate.academy.service.ParserService;
+
+public class ParserServiceImpl implements ParserService {
+    private final String dataSeparator;
+    private final int operationCodeIndex;
+    private final int quantityIndex;
+    private final int fruitNameIndex;
+
+    public ParserServiceImpl() {
+        dataSeparator = ",";
```

Comment:
No need to initialize them since they're all constants.

`private static final String SEP(or SEPARATOR) = ",";`
...